migrations/versions/94d2839daa08_.py

```diff
@@ -151,7 +151,7 @@ def upgrade():
     sa.Column('kind', sa.String(length=255), nullable=False),
     sa.ForeignKeyConstraint(['backup_id'], ['backup.id'], name=op.f('fk_message_backup_id_backup')),
     sa.PrimaryKeyConstraint('id', name=op.f('pk_message')),
-    mysql_row_format='COMPRESSED'
+    mysql_row_format=os.getenv('DB_ROW_FORMAT', 'COMPRESSED')
```

Comment:
It would be nice to store `DB_ROW_FORMAT` in the `settings/*.py` files and use `app.config` here rather than `os.getenv`. This looks to be related: https://stackoverflow.com/questions/49409691/flask-sqlalchemy-acess-different-schema

Reply:
I added this for the wrong  file. This comment is referring to the change in `models.py` line 962.